packages/dcos-history/extra/dcos-history.service

```diff
@@ -8,7 +8,8 @@ Restart=always
 StartLimitInterval=0
 RestartSec=5
 EnvironmentFile=/opt/mesosphere/environment
-EnvironmentFile=-/run/dcos/etc/history-service/history-service.env
```

Comment:
Pretty sure the extension still needs runtime extension and not just config extension.